src/migration/02172023/migrate-groups-dynamodb-02172023.migration-test.ts

```diff
@@ -225,16 +226,26 @@ describe("Test migration", () => {
 const computeProperties = (data: FetchedData): Properties => {
   const valueDistribution: { [tier: number]: number } = {};
   let accountsNumber = 0;
+  let minValue: BigNumberish = ethers.constants.MaxUint256;
```

Comment:
let just put null by default

Reply:
But a BigNumberish can't be null, so `minValue` and `maxValue` will be of type: `BigNumberish | null`
that's why I thought put 0 and MaxUint256 is better.